conda_env/__main__.py

```diff
@@ -1,7 +0,0 @@
-# Copyright (C) 2012 Anaconda, Inc
-# SPDX-License-Identifier: BSD-3-Clause
-import sys
-
-from .cli.main import main
-
-sys.exit(main())
```

Comment:
Must have been for `python -m conda_env` if anyone uses that

Reply:
Restored file and added deprecation warning